examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
+
+# Reset the entrypoint
+ENTRYPOINT ["/bin/sh", "-c"]
+
+RUN yum install -y clamav-server clamav-scanner clamav-data --disablerepo=origin-local-release \
+      && mkdir -p /image-data && chmod 0777 /image-data
```

Comment:
Let's also remember that there is another degree of freedom: image-inspector as a library. If we design the code correctly then we wouldn't prevent anyone from building a long-running node agent `image-inspector-daemon` (of course it will need an extra amount of orchestration).
Maybe the daemon could be specialized only to scan existing containers so that it won't have to deal with storage management.
Anyway overall I still like the job-based approach, but I think we should facilitate anyone who would like to take a different approach.